certified-connectors/AzureDigitalTwins/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AzureDigitalTwins",
+    "description": "Azure Digital Twins",
+    "version": "1.0"
+  },
+  "host": "digitaltwins.azure.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/digitaltwins/{twinid}": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@sankumrav using only default responses with defined schema is not allowed. Please refer to [information about responses](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-exact-operation-responses) and to [coding standards](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).